src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,31 @@
 package core.basesyntax;
 
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.figurehandlers.FigureSupplier;
 /**
  * Feel free to remove this class and create your own.
  */
+
 public class HelloWorld {
 
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
+    private static int ARRAY_LENGTH = 6;
```

Comment:
this is not an informative constant name - what array? 

it would be better to name it like `NUMBER_OF_FIGURES`